index.js

```diff
@@ -34,11 +34,12 @@ var split = String.prototype.split;
 var delimiter = '||||';
 
 var russianPluralGroups = function (n) {
-  var end = n % 10;
-  if (n !== 11 && end === 1) {
+  var num = n % 100;
```

Comment:
i call this lastTwo in other plural functions

Reply:
Renamed.
Please recheck.
Initially I made just one change - `n = n % 100` as we don't need anything else here. But your linter got upset :)

Reply:
:-) that's fine.

We still need tests for the other languages tho.